index.html

```diff
@@ -24,24 +29,23 @@
 
   <div id="drinks-view" class="container">
     <!-- EXAMPLE CARD -->
-    <!-- <div class="card">
-      <div class="card-image waves-effect waves-block waves-light">
-        <img class="drink-image"
-          src="assets/images/__opt__aboutcom__coeus__resources__content_migration__liquor__2018__09__04153106__mojito-720x720-recipe-a55b114fc99c4a64b38c9ef6d1660e20.jpg">
-      </div>
-      <div class="card-content">
-        <span class="card-title activator grey-text text-darken-4">Mojito<i class="material-icons right"></i></span>
-        <p><a href="#">This is a link</a></p>
-      </div>
-      <div class="card-reveal">
-        <span class="card-title grey-text text-darken-4">Mojito Recipe<i class="material-icons right">X</span>
-        <p>Place mint leaves and 1 lime wedge into a sturdy glass. Use a muddler to crush the mint and lime to release
-          the mint oils and lime juice. Add 2 more lime wedges and the sugar, and muddle again to release the lime
-          juice. Do not strain the mixture. Fill the glass almost to the top with ice. Pour the rum over the ice, and
-          fill the glass with carbonated water. Stir, taste, and add more sugar if desired. Garnish with the remaining
-          lime wedge.</p>
-      </div>
-    </div> -->
+    <!-- <div class="row">
```

Comment:
I think I'll need the "drinks-view" id hook to be right before the cards that I'm generating, so it may have to be here instead of on the container otherwise we'll be generating a new row each time a card is generated.